packages/SwingSet/src/controller.js

```diff
@@ -296,7 +285,8 @@ export async function buildVatController(config, withSES = true, argv = []) {
     vatAdminVatSetup: await evaluateToSetup(ADMIN_VAT_PATH, 'vat-vatAdmin'),
   };
 
-  const kernel = buildSESKernel(sesEvaluator, kernelEndowments);
+  const kernelSetup = await evaluateToSetup(KERNEL_PATH, 'kernel');
```

Comment:
`evaluateToSetup` is now misnamed, if we're also using it to load the kernel (which doesn't provide a `setup` function)

Reply:
Agreed.